patlub/bucketList
# Adds session feature and refactor activity to items

**What does the PR do?**

Adds session feature to the app and also adds a description to the bucket. Plus it refactors the activity to Item.

**Where should the reviewer start**

A few changes to functions in BucketList.py

**How should this be manually tested by a user**

The user should be to add and edit a description to a bucket

@CeciliaCaroline @skipcheru
